Improve class comments (#126)
hpi-swa-teaching/ChangesBrowser: packages/ChangesReloaded-Core.package/CRChangeListItem.class/README.md

> A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state.

Reviewer: ```suggestion
A CRChangeListItem is an item of the CRChangeList that represents a ChangeRecord in the CRChangeList and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```

Reviewer 2: "A CRChangeListItem is an item **in** the…"
It's a shame there's no suggestion for suggestions

Instance Variables

> 	changeRecord			represented ChangeRecord as actual Squeak ChangeRecord

Reviewer: ```suggestion
	changeRecord			actual ChangeRecord wrapped by CRChangeListItem
```

> 	isPicked					wether a specific represented ChangeRecord is picked for loading

Reviewer: ```suggestion
	isPicked					true if the wrapped ChangeRecord is picked for getting loaded into the image
```

> A CRChangeListItem is an item of the CRBrowser's change list that represents a ChangeRecord and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.

Reviewer 2: ```suggestion
A CRChangeListItem is an item in the CRBrowser's change list that represents a ChangeRecord and has a selection state. It wraps a ChangeRecord of the Squeak image and can be handled as such, all messages not understood by the CRChangeListItem are forwarded to the wrapped ChangeRecord.
```
